Source/WTF/wtf/URL.cpp

```diff
@@ -269,7 +269,7 @@ URL URL::truncatedForUseAsBase() const
     return URL(m_string.left(m_pathAfterLastSlash));
 }
 
-#if !USE(CF)
+#if !USE(CF) || PLATFORM(WIN)
```

Comment:
This change should not be needed, because `USE(CF)` should not ever be true on `PLATFORM(WIN)` any more.

Reply:
Apparently I am mistaken. The WinCairo port is using `USE(CF)` with CFLite on Windows. I wonder if that’s a needed long term strategy. Does not seem to be helpful long term to use CFLite.